# Assorted Computational Genomics Data Visualizations


**Introduction** 

This repository contains Biplots, Volcano plots, PCA plots, Heatmaps and more [Computational Genomics](https://en.wikipedia.org/wiki/Computational_genomics) data created and visualized during the University of Pittsburgh course, Computational Biology (BIOSC1540), with [Dr. Miler Lee](https://mtleelab.pitt.edu/). 

*Note: The following visualizations were created using a combination of Bash and R* 

**Files** 

:page_facing_up: `Miscellaneous Visualizations/ Assorted Visualizations.pdf` 

Contains [Differential Gene Expression](https://www.ebi.ac.uk/training/online/course/functional-genomics-ii-common-technologies-and-data-analysis-methods/differential-gene) Biplot, Volcano plot, [Sequence Motif](https://en.wikipedia.org/wiki/Sequence_motif) output from [ChIP-Seq](https://www.illumina.com/techniques/sequencing/dna-sequencing/chip-seq.html) analysis of MXTX2 transcription factor on Zebrafish embryos and [Comparative Genomics](https://en.wikipedia.org/wiki/Comparative_genomics) analysis of genes between humans and gorillas using [UCSC LiftOver](https://genome.ucsc.edu/cgi-bin/hgLiftOver). 


:page_facing_up: `Heatmaps/ Differential Gene Expression Replicate 1.pdf` and `Heatmaps/ Differential Gene Expression Replicate 2.pdf`

*Note: Ignore the title of the plots themselves since these were unique identifier strings assigned to students of the course.*

[Heatmap](https://www.optimizely.com/optimization-glossary/heatmap/) created using [High-Dimensional Data Reduction](https://en.wikipedia.org/wiki/Dimensionality_reduction) and Visualization techniques to show the gene expression levels of a Zebrafish embryo for each gene it has at different time points. 

:page_facing_up: `Principal Component Analysis/ PC1 VS PC2.pdf` and `Principal Component Analysis/ PC2 VS PC3.pdf`

*Note: Ignore the title of the plots themselves since these were unique identifier strings assigned to students of the course.*

[Principal Component Analysis (PCA)](http://setosa.io/ev/principal-component-analysis/) plots created from NGS data of Zebrafish embryos. The two graphs are showing different Principal Components being mapped from the NGS data. 

:page_facing_up: `Miscellaneous Visualizations/ RPKM_VS_TPM.pdf` 

Visual comparison created to show the relationship between the values of NGS read quantification [normalization](http://www.people.vcu.edu/~mreimers/OGMDA/normalize.expression.html) using methods of [Reads-per-Kilobase Million (RPKM) and Transcripts Per Million (TPM)](https://www.rna-seqblog.com/rpkm-fpkm-and-tpm-clearly-explained/) 



**Contact Information** 

![interests](https://avatars1.githubusercontent.com/u/38919947?s=400&u=49ab1365a14fac78a91e425efd583f7a2bcb3e25&v=4)

Yogindra Raghav (YogiOnBioinformatics) 

yraghav97@gmail.com
